src/tools/sidctl/sidctl.c

```diff
@@ -329,36 +159,36 @@ static int _usid_cmd_stats(struct args *args, output_format_t format)
 	return r;
 }
 
-static int _usid_cmd_version(struct args *args, output_format_t format)
+static int _usid_cmd_version(struct args *args, output_format_t format, struct buffer *outbuf)
 {
-	struct buffer *         buf = NULL;
+	struct buffer *         readbuf = NULL;
 	struct usid_msg_header *hdr;
 	size_t                  size;
 	struct usid_version *   vsn = NULL;
 	int                     r;
-	r = usid_req(LOG_PREFIX, USID_CMD_VERSION, 0, NULL, NULL, &buf);
+	r = usid_req(LOG_PREFIX, USID_CMD_VERSION, 0, NULL, NULL, &readbuf);
 
-	print_start_document(format, 0);
+	print_start_document(format, outbuf, 0);
 
-	print_uint_field(KEY_SIDCTL_PROTOCOL, USID_PROTOCOL, format, true, 1);
-	print_uint_field(KEY_SIDCTL_MAJOR, SID_VERSION_MAJOR, format, true, 1);
-	print_uint_field(KEY_SIDCTL_MINOR, SID_VERSION_MINOR, format, true, 1);
-	print_uint_field(KEY_SIDCTL_RELEASE, SID_VERSION_RELEASE, format, r == 0, 1);
+	print_uint_field(KEY_SIDCTL_PROTOCOL, USID_PROTOCOL, format, outbuf, true, 1);
+	print_uint_field(KEY_SIDCTL_MAJOR, SID_VERSION_MAJOR, format, outbuf, true, 1);
+	print_uint_field(KEY_SIDCTL_MINOR, SID_VERSION_MINOR, format, outbuf, true, 1);
+	print_uint_field(KEY_SIDCTL_RELEASE, SID_VERSION_RELEASE, format, outbuf, r == 0, 1);
 
 	if (r == 0) {
-		buffer_get_data(buf, (const void **) &hdr, &size);
+		buffer_get_data(outbuf, (const void **) &hdr, &size);
```

Comment:
When I tried `sidctl version`, I got incorrect output for SID version - it's just this line that has typo, I think. It should be `readbuf` instead of `outbuf` here.

Reply:
Thanks.  Fixed.